clean/lake_charles_pd_cprr.py

```diff
@@ -69,23 +141,23 @@ def split_rows_with_multiple_officers(df):
     return df
 
 
-def drop_rows_missing_disp_charges_and_action(df):
+def drop_rows_missing_disp_charges_and_action_20(df):
     return df[~((df.disposition == '') & (df.charges == '') & (df.action == ''))]
 
 
-def assign_empty_first_name_column(df):
+def assign_empty_first_name_column_20(df):
     df.loc[:, 'first_name'] = ''
     return df
 
 
-def review_first_names_from_post():
-    df = pd.read_csv(data_file_path('match/lake_charles_pd_cprr_2020_extracted_first_names.csv'))
+def review_first_names_from_post_20():
+    df = pd.read_csv(data_file_path('match/lake_charles_pd_cprr_2020_assign_first_name_from_post_pprr_2020_11_06.xlsx'))
```

Comment:
Excel libraries in Python have the tendency to parse numerical and date/time values in strange ways which we have no control over. To ensure precise processing, we should only accept CSV as input.

Reply:
Also, `make` might not be able to pick up input files other than CSVs.

Reply:
Never mind, this is not part of the processing, is it? Why not just put this in a notebook? A clean script importing things from the "match" directory seems backward to me.

Reply:
Also, why do you need to view it in a data frame? I thought the Excel format makes it easy to review in Excel?